.github/workflows/tools.yaml

```
@@ -178,10 +178,11 @@ jobs:
         repository-list: ${{ needs.setup-ci-tools.outputs.repository-list }}
         tool-list: ${{ needs.setup-ci-tools.outputs.tool-list }}
```

```suggestion
        tool-list: ${{ needs.setup-ci-tools.outputs.tool-list }}
      continue-on-error: true
```
This way the step can fail but has no impact on the status of the job, which we control with the grep below.